Create Executive copy - February 22, 2024.md (#2565)
makerdao/community: governance/votes/Executive vote - February 22, 2024.md

---

> title: Template - [Executive Vote] Stability Fee Changes, Spark Protocol DC-IAM Parameter Changes, Housekeeping Actions, Spark Proxy Spell - February 22, 2024

Reviewer: We should use reader-friendly terms whenever possible and avoid technical terms.

```suggestion
title: Template - [Executive Vote] Stability Fee Changes, Spark Protocol D3M Parameter Changes, Housekeeping Actions, Spark Proxy Spell - February 22, 2024
```

> title: Template - [Executive Vote] Stability Fee Changes, Spark Protocol DC-IAM Parameter Changes, Housekeeping Actions, Triggering Spark Proxy Spell - February 22, 2024

Reviewer: I'll try to stick to titling the items in similar ways to how we've done it before for consistency

```suggestion
title: Template - [Executive Vote] Stability Fee Changes, Spark Protocol DC-IAM Parameter Changes, Housekeeping Actions, Spark Proxy Spell - February 22, 2024
```

summary: Stability Fee changes for ETH-A, ETH-B, ETH-C, WSTETH-A, WSTETH-B, WBTC-A, WBTC-B, WBTC-C, Spark Protocol DC-IAM parameter changes, pushing USDP out of input conduit, yanking BA Labs DAI and MKR vest streams, triggering Spark proxy spell.  
date: 2024-02-22T00:00:00.000Z
address: "$spell_address"

---

> # [Executive Vote] Stability Fee Changes, Spark Protocol DC-IAM Parameter Changes, Increasing wstETH Supply Cap on Spark Ethereum, Increasing the Spark DAI Effective Borrow APY, Pushing USDP out of Input Conduit, Yanking Vest Streams, Triggering Spark Proxy Spell - February 22, 2024

Reviewer: This should match the title. Please, check the Executive Copy template: https://github.com/makerdao/community/blob/master/governance/votes/templates/Executive%20vote%20-%20%24date_MONTH_DD%2C_YYYY.md

```suggestion
# [Executive Vote] Stability Fee Changes, Spark Protocol DC-IAM Parameter Changes, Housekeeping Actions, Spark Proxy Spell - February 22, 2024
```

The Governance Facilitators, Sidestream, and Dewiz have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.

If you are new to voting in the Maker Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.

---

## Executive Summary

If this executive proposal passes, the following actions will occur within the Maker Protocol:

> - ETH-A, ETH-B, ETH-C, WSTETH-A, WSTETH-B, WBTC-A, WBTC-B, WBTC-C [Stability Fee](https://manual.makerdao.com/parameter-index/vault-risk/param-stability-fee) decreases, as detailed below. 

Reviewer: ```suggestion
- ETH-A, ETH-B, ETH-C, WSTETH-A, WSTETH-B, WBTC-A, WBTC-B, WBTC-C [Stability Fees](https://manual.makerdao.com/parameter-index/vault-risk/param-stability-fee) will be decreased, as detailed below. 
```

- The following housekeeping actions will be carried out:
  - BA Labs DAI and MKR vest streams will be yanked, as detailed below.
  - USDP will be pushed out of the input conduit, as detailed below.
- A Spark Protocol proxy spell will be triggered, as detailed below.

**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**

Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.

If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.

---

## Proposal Details

### ETH-A, ETH-B, ETH-C, WSTETH-A, WSTETH-B, WBTC-A, WBTC-B, WBTC-C [Stability Fee](https://manual.makerdao.com/parameter-index/vault-risk/param-stability-fee) Decreases

> **Authorization:** [Ecosystem Approval](http://forum.makerdao.com/t/stability-scope-parameter-changes-9/23688/2)

Reviewer: Please, add a backslash to generate a linebreak at the end of the authorization line.

```suggestion
**Authorization:** [Ecosystem Approval](http://forum.makerdao.com/t/stability-scope-parameter-changes-9/23688/2)\
```

- Increase the DIRECT-SPARK-DAI [Maximum Debt Ceiling (`line`)](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) by 300 million DAI from 1.2 billion DAI to **1.5 billion DAI**.
- Increase the DIRECT-SPARK-DAI [Target Available Debt (`gap`)](https://manual.makerdao.com/module-index/module-dciam#target-available-debt-gap) by 20 million DAI from 20 million DAI to **40 million DAI**.
- Increase the DIRECT-SPARK-DAI [Ceiling Increase Cooldown (`ttl`)](https://manual.makerdao.com/module-index/module-dciam#ceiling-increase-cooldown-ttl) by 12 hours from 12 hours to **24 hours**.
- Increase the DIRECT-SPARK-DAI buffer by 20 million DAI from 30 million DAI to **50 million DAI**.

### Housekeeping Actions

**Authorization:** [Ecosystem Approval](https://forum.makerdao.com/t/proposed-housekeeping-items-upcoming-executive-spell-2024-02-22/23697/2), [2](http://forum.makerdao.com/t/proposed-housekeeping-items-upcoming-executive-spell-2024-02-22/23697/4)\
**Proposal:** [Forum post "Proposed Housekeeping items (Upcoming Executive Spell - 2024-02-22)"](https://forum.makerdao.com/t/proposed-housekeeping-items-upcoming-executive-spell-2024-02-22/23697)

> #### Push USDP out of the Input Conduit

Reviewer 2: **nit** - align naming convention with most recent copy.

```suggestion
#### USDP Input Conduit Management
```

If this executive proposal passes, following the recommendation of the Governance Facilitators in collaboration with the spell team, the following operations will be performed:

> - Raise `PSM-PAX-A` DC to **1,000,000 DAI**

Reviewer 2: **nit** - PSM-PAX-A has been formatted as code (`PSM-PAX-A`) but DIRECT-SPARK-DAI and core vaults above have not. Either option is fine but it should probably be consistent within the same doc. We would lean toward not formatting vaults and contract names as code, but instead using it for contract calls. See our formatting for the recent copy - https://github.com/makerdao/community/blob/master/governance/votes/Executive%20vote%20-%20February%207%2C%202024.md#gusd-input-conduit-management

Ultimately, however, this is a nit and non-blocking.

> - Call push() on `MCD_PSM_PAX_A_INPUT_CONDUIT_JAR` (use push(uint256 amt)) to push **754,598.72 USDP**

Reviewer 2: :yellow_circle: **Change requested**

`push()` and `push(uint256 amt)` are different functions on the contract. We are calling the latter.

source: https://github.com/makerdao/rwa-toolkit/blob/master/src/conduits/RwaSwapInputConduit2.sol#L230-L240

```suggestion
- Call push(uint256 amt) on `MCD_PSM_PAX_A_INPUT_CONDUIT_JAR` with an `amt` of **754,598.72 USDP**
```

> - Raise `PSM-PAX-A` DC to **1,000,000 DAI**
> - Call push() on `MCD_PSM_PAX_A_INPUT_CONDUIT_JAR` (use push(uint256 amt)) to push **754,598.72 USDP**
> - Call void() on `MCD_PSM_PAX_A_JAR`
> - Set `PSM-PAX-A` DC to **0 DAI**
> 
> This will have the following net results:
> 
> - **Debt Ceiling** remains at **0 DAI** after the transaction has been executed.

Reviewer 2: **nit** - inconsistent use of DC vs Debt Ceiling

```suggestion
- Raise `PSM-PAX-A` Debt Ceiling to **1,000,000 DAI**
- Call push() on `MCD_PSM_PAX_A_INPUT_CONDUIT_JAR` (use push(uint256 amt)) to push **754,598.72 USDP**
- Call void() on `MCD_PSM_PAX_A_JAR`
- Set `PSM-PAX-A` Debt Ceiling to **0 DAI**

This will have the following net results:

- **Debt Ceiling** remains at **0 DAI** after the transaction has been executed.
```

- Dai generated from `PSM_PAX_A` increases by **754,598.72 DAI**.
- USDP balance of `MCD_JOIN_PSM_PAX_A` increases by **754,598.72 USDP**.
- USDP balance of `PSM_PAX_A_INPUT_CONDUIT_JAR` decreases to **0 USDP**.
- Surplus Buffer increases by **754,598.72 DAI**.

#### Yank BA Labs DAI and MKR Vest Streams

> If this executive proposal passes, following the recommendation of the Governance Facilitators, the BA Labs DAI and MKR vest streams will be canceled (yanked), namely:

Reviewer 2: **nit** - consider adding that funding will switch to Launch Project as we anticipate that someone will ask if it isn't specified within the copy.

- yank BA Labs **DAI** stream **20**.
- yank BA Labs **DAI** stream **21**.
- yank BA Labs **MKR** stream **35**.

### Trigger Spark Proxy Spell

**Authorization:** [Ecosystem Approval](https://forum.makerdao.com/t/stability-scope-parameter-changes-9/23688/2), [Governance Poll](https://vote.makerdao.com/polling/QmQC1UXZ)\
**Proposal:** [Forum post "Stability Scope Parameter Changes #9"](https://forum.makerdao.com/t/stability-scope-parameter-changes-9/23688), [Forum post "[Feb 14, 2024] Proposed Changes to SparkLend for Upcoming Spell"](https://forum.makerdao.com/t/feb-14-2024-proposed-changes-to-sparklend-for-upcoming-spell/23684)

If this executive proposal passes, the SparkLend Proxy Spell (Ethereum) at TBD will be executed, containing the following major items:

> - Increase the [wstETH](https://app.spark.fi/reserve-overview/?underlyingAsset=0x7f39c581f595b53c5cb19bd0b3f8da6c935e2ca0&marketName=proto_spark_v3) [Supply Cap](https://docs.sparkprotocol.io/developers/sparklend/features/supply-borrow-caps#supply-caps) to **1,2M**.

Reviewer 2: :yellow_circle: **Change requested**

1. Change to decimal point
2. Add unit
3. Specify "million"

```suggestion
- Increase the [wstETH](https://app.spark.fi/reserve-overview/?underlyingAsset=0x7f39c581f595b53c5cb19bd0b3f8da6c935e2ca0&marketName=proto_spark_v3) [Supply Cap](https://docs.sparkprotocol.io/developers/sparklend/features/supply-borrow-caps#supply-caps) to **1.2 million wstETH**.
```

---

> title: Template - [Executive Vote] Stability Fee Changes, Spark Protocol DC-IAM Parameter Changes, Increasing wstETH Supply Cap on Spark Ethereum, Increasing the Spark DAI Effective Borrow APY, Housekeeping Actions, Triggering Spark Proxy Spell - February 22, 2024

Reviewer: In past Executives, we've never included the SubDao (Spark) changes in the title, summary, or details. They always fall under the Spark Proxy Spell section

```suggestion
title: Template - [Executive Vote] Stability Fee Changes, Spark Protocol DC-IAM Parameter Changes, Housekeeping Actions, Triggering Spark Proxy Spell - February 22, 2024
```

> summary: Stability Fee changes for ETH-A, ETH-B, ETH-C, WSTETH-A, WSTETH-B, WBTC-A, WBTC-B, WBTC-C, Spark Protocol DC-IAM parameter changes, increasing wstETH supply cap to 1.2M on Spark Ethereum, increasing the Spark DAI effective borrow APY to 6.70%, pushing USDP out of input conduit, yanking BA Labs DAI and MKR vest streams, triggering Spark proxy spell.  

Reviewer: ```suggestion
summary: Stability Fee changes for ETH-A, ETH-B, ETH-C, WSTETH-A, WSTETH-B, WBTC-A, WBTC-B, WBTC-C, Spark Protocol DC-IAM parameter changes, pushing USDP out of input conduit, yanking BA Labs DAI and MKR vest streams, triggering Spark proxy spell.  
```

date: 2024-02-22T00:00:00.000Z
address: "$spell_address"

---
# [Executive Vote] Stability Fee Changes, Spark Protocol DC-IAM Parameter Changes, Increasing wstETH Supply Cap on Spark Ethereum, Increasing the Spark DAI Effective Borrow APY, Pushing USDP out of Input Conduit, Yanking Vest Streams, Triggering Spark Proxy Spell - February 22, 2024

The Governance Facilitators, Sidestream, and Dewiz have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.

If you are new to voting in the Maker Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.

---

## Executive Summary

If this executive proposal passes, the following actions will occur within the Maker Protocol:

- ETH-A, ETH-B, ETH-C, WSTETH-A, WSTETH-B, WBTC-A, WBTC-B, WBTC-C [Stability Fee](https://manual.makerdao.com/parameter-index/vault-risk/param-stability-fee) decreases, as detailed below. 
- Spark Protocol [DC-IAM](https://manual.makerdao.com/module-index/module-dciam) parameter changes, as detailed below.

> - Increasing wstETH Supply Cap to **1.2M** on Spark Ethereum, as detailed below.

Reviewer: ```suggestion
```

> - Increasing the Spark DAI Effective Borrow APY by 0.24 percentage points, from 6.46% to **6.70%**, as detailed below.

Reviewer: ```suggestion
```
